src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+
+  const splittedSource = String(sourceString).split(';');
```

Comment:
Consider handling cases where `sourceString` might be empty or not contain any valid key-value pairs. This could prevent unnecessary processing and potential errors.